config_auth.go

```diff
@@ -196,10 +197,8 @@ func (c *AuthConfig) ValidateAndDefault() error {
 		}
 	}
 
-	if c.Github != nil {
-		if c.Github.Users == nil && c.Github.Organization == "" {
-			catcher.Add(errors.New("must specify either a set of users or an organization for GitHub authentication"))
-		}
+	if c.Github != nil && c.Github.AppId == 0 && c.Github.Users == nil && c.Github.Organization == "" {
+		catcher.Add(errors.New("must specify either a set of users or an organization or an app id for GitHub authentication"))
```

Comment:
nit: You can simplify this down to `catcher.New`.

Reply:
done